streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -288,6 +291,8 @@ public StreamThread(TopologyBuilder builder,
         this.standbyRecords = new HashMap<>();
 
         this.stateDirectory = new StateDirectory(applicationId, config.getString(StreamsConfig.STATE_DIR_CONFIG), time);
+        Object maxPollInterval = consumerConfigs.get(ConsumerConfig.MAX_POLL_INTERVAL_MS_CONFIG);
+        this.rebalanceTimeoutMs =  maxPollInterval == null ? 300000 : (Integer) ConfigDef.parseType(ConsumerConfig.MAX_POLL_INTERVAL_MS_CONFIG, maxPollInterval, Type.INT);
```

Comment:
Could you please point me to the code where we load all default consumerConfigs inside `StreamsConfig `. As far as I am able to see is that to consumer we pass any values that we override in streamsConfig. The defaults of `ConsumerConfig` are intenal to consumers only. That is why I had to copy the default value for `ConsumerConfig.MAX_POLL_INTERVAL_MS_CONFIG` in case streamsConfig did not specify one.